src/main/java/teammates/common/util/Const.java

```diff
@@ -1498,14 +1498,24 @@ private Const() {
         public static final String ROLE_UNKNOWN = "Unknown";
         public static final String ROLE_MASQUERADE_POSTFIX = "(M)";
 
-        public static final String AUTH_NOT_LOGIN = "Unregistered";
+        public static final String PREFIX_STUDENT_PAGE = "student";
+        public static final String PREFIX_INSTRUCTOR_PAGE = "instructor";
+        public static final String PREFIX_AUTO_PAGE = "/auto";
+
+        public static final String AUTH_NOT_LOGIN = "Unknown";
 
         public static final String FIELD_SEPARATOR = "|||";
+        public static final String FIELD_CONNECTOR = "%";
 
-        public static final String TEST_DATA_POSTFIX = ".tmt";
+        public static final String TIME_FORMAT_LOGID = "yyyyMMddHHmmssSS";
+
+        public static final String TESTING_DATA_EMAIL_POSTFIX = ".tmt";
 
         public static final int TIME_TAKEN_EXPECTED = 10000;
         public static final int TIME_TAKEN_MODERATE = 20000;
+
+        public static final String MESSAGE_ERROR_ACTION_NAME = "Error when getting ActionName for requestUrl : %1$s";
+        public static final String MESSAGE_ERROR_LOG_MESSAGE_FORMAT = "Log message format not as expected: %1$s";
     }
```

Comment:
@damithc I know the concern. I try to separate the commit and make it more cohesive but ends with just one big commit with everything. The thing is that we have totally change the generation of log message. We now have a generator to make log messages for different actions. Compare to the old way (multiples constructor), it's much better in term of code quality and code reuse. Therefore, when updated, we need to change everywhere. 

The test cases and log code has been separated. Also,  `Introduce LogMessageGenerator and update ActivityLogEntry accordingly`, `Update all Action to use LogMessageGenerator`, `	Update affected ActionTest as LogMessageGenerator standardize the way to generate logs.` should be one single commit if we want to make it cohesive. I think separate them in this way (although there are not cohesive) will make the review easier.